Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3068,28 +3068,28 @@ void WebPageProxy::updateTouchEventTracking(const WebTouchEvent& touchStartEvent
     const EventNames& names = eventNames();
     for (auto& touchPoint : touchStartEvent.touchPoints()) {
         IntPoint location = touchPoint.location();
-        auto updateTrackingType = [this, location](TrackingType& trackingType, const AtomString& eventName) {
+        auto updateTrackingType = [this, location](TrackingType& trackingType, EventTrackingRegions::Event eventName) {
```

Comment:
Let's also change the variable name from `eventName` to `event`.

Reply:
Done.